packages/liveblocks-client/src/AbstractCrdt.ts

```diff
@@ -20,41 +20,41 @@ export interface Doc {
 }
 
 export abstract class AbstractCrdt {
-  #parent?: AbstractCrdt;
-  #doc?: Doc;
-  #id?: string;
-  #parentKey?: string;
+  private __parent?: AbstractCrdt;
+  private __doc?: Doc;
+  private __id?: string;
+  private __parentKey?: string;
```

Comment:
The rationale for this, as explained in #87, makes a lot of sense 👍

Reply:
Do you think it’s worth to set up a custom ESLint rule for this, so future devs will automatically get a rejection when using private fields/methods, and see this explanation as the reason?

When publishing `decoders` v2.0.0, I forbid myself to use specific costly Babel constructs going forward, and got a [big drop in bundle size](https://github.com/nvie/decoders/blob/c3d0b413ac7f06adab8a83ee26ad97b169724831/docs/assets/size-comparison%402x.png) from it.

I enforced this upon myself by setting up an [ESLint rule](https://github.com/nvie/decoders/blob/51abb4050216d7fa96ba4e7c4dc77d42d4d5b7b7/.eslintrc.js#L75-L81) that forbids the use of the `for..of` loops in library code, and replaced those with `.forEach()` in most cases. A bit uglier, of course, but users benefit from smaller bundles (and a faster runtime).

Codifying this decision hugely helps enforce this in the future, and acts as a reminder of the rationale for the disallowance at the same time, because the reason is explained in the error message.

Reply:
I like the idea of using Eslint rules to prevent some features that would increase the final bundle size! Definitely worth it as the team is growing.